vscode-lean4/src/utils/fsHelper.ts

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+import { readFile, promises, PathLike } from 'fs';
+
+
+export async function fileExists(pathFile: PathLike): Promise<boolean> {
+    /*
+    Helper used to replace fs.existsSync (using existsSync to check for the existence
+    of a file before calling fs.open(), fs.readFile() or fs.writeFile() is not recommended.
+    Doing so introduces a race condition, since other processes may change the file's state between the two calls.
+    Instead, user code should open/read/write the file directly and handle the error raised if the file does not exist.)
+    param: pathFile - A string representing a PathLike
+
+    returns Promise<boolean> a boolean that represents if a file exist
```

Comment:
```suggestion
```

No need to duplicate the type annotation.

Reply:
Ok, resolved! Thanks!!